src/function_lua.c

```diff
@@ -479,7 +479,7 @@ int luaEngineInitEngine() {
                             "    i = dbg.getinfo(3,'nSl')\n"
                             "  end\n"
                             "  if type(err) ~= 'table' then\n"
-                            "    err = {err='ERR' .. tostring(err)}"
+                            "    err = {err='ERR ' .. tostring(err)}"
```

Comment:
I'd like to use this opportunity to mention again that I think we should avoid code duplication when defining `errh_func` in both `luaEngineInitEngine()` and `scriptingInit()`. Maybe this PR can fix that as well?

Reply:
I don't have a strong opinion since I don't understand this code well, but I'll concede that it took me a long time to figure out this was the error, the documentation and code was a bit hard to follow. So if there is a good "natural" place to put this, I'm fine with that.

Reply:
@MeirShpilraien :point_up:

Reply:
@yoav-steinberg I agree its should be merge and I am currently working on it along side some other fixes, so not need to do duplicate work :)